frontend/app/framework/angular/pipes/date-time.pipes.spec.ts

```diff
@@ -98,6 +99,20 @@ describe('FromNowPipe', () => {
         expect(actual).toBe(expected);
     });
 
+    it('should format to from now string in Dutch', () => {
+        try {
+            DateHelper.setlocale('nl');
+            const pipe = new FromNowPipe();
+
+            const actual = pipe.transform(DateTime.now().addMinutes(-4));
+            const expected = '4 minuten';
+
+            expect(actual).toBe(expected);
+          } finally {
```

Comment:
Something is broken with formatting here. in VSCode you can just right click and choose "format code" or so...

Reply:
I keep missing those formatting errors, I need to check my console more.

I’m used that it’s fixed on compiling. 
But I fix it.